openach/templatetags/templatehelpers.py

```diff
@@ -0,0 +1,16 @@
+from django import template
+
+register = template.Library()
+
+
+@register.simple_tag
+def relative_url(value, field_name, urlencode=None):
+    url = "?{}={}".format(field_name, value)
+    if urlencode:
+        querystring = urlencode.split("&")
```

Comment:
For URLs, you don't want to be doing string manipulation. Instead use Python's URL manipulation libraries: https://docs.python.org/3/library/urllib.parse.html#urllib.parse.parse_qs